src/Ads/AssetSuggestionsService.php

```diff
@@ -187,17 +194,48 @@ protected function get_asset_group_asset_suggestions( int $id, string $type ): a
 	/**
 	 * Get assets from specific post or term.
 	 *
-	 * @param int    $id Post or Term ID.
+	 * @param int    $id Post or Term ID, or self::HOMEPAGE_KEY_ID.
 	 * @param string $type Only possible values are post or term.
 	 *
-	 * @return array All assets available for specific term or post.
+	 * @return array All assets available for specific term, post or homepage.
 	 */
 	protected function get_wp_assets( int $id, string $type ): array {
 		if ( $type === 'post' ) {
 			return $this->get_post_assets( $id );
+		} elseif ( $type === 'term' ) {
+			return $this->get_term_assets( $id );
+		} else {
+			return $this->get_homepage_assets();
 		}
 
-		return $this->get_term_assets( $id );
+	}
+
+	/**
+	 * Get assets from the homepage.
+	 *
+	 * @return array Assets available for the homepage.
+	 * @throws Exception If the homepage id is invalid.
+	 */
+	protected function get_homepage_assets(): array {
+		$home_page = $this->wp->get_static_homepage();
+
+		// Static homepage.
+		if ( $home_page ) {
+			return $this->get_post_assets( $home_page->ID );
+		}
+		// Non static homepage.
```

Comment:
💅  I guess those are default values. Since now static homepage will be potentially overrided with the Asset Group with the same URL